pymeasure/adapters/prologix.py

```diff
@@ -54,7 +56,18 @@ class PrologixAdapter(VISAAdapter):
 
     :param kwargs: Key-word arguments if constructing a new serial object
 
-    :ivar address: Integer GPIB address of the desired instrument
+    :ivar address: Integer GPIB address of the desired instrument.
+
+    Usage example:
+
+    .. code::
+
+        adapter = PrologixAdapter("ASRL5::INSTR", 7)
+        # generate another instance with a different GPIB address:
+        adapter2 = adapter.gpib(9)
+        sourcemeter1 = Keithley2400(adapter)  # at GPIB address 7
+        sourcemeter2 = Keithley2400(adapter2)  # at GPIB address 9
```

Comment:
I would consider using another instrument class for the second instrument at address 9 to avoid giving the impression, to the casual reader, that you can only use multiple instruments of the same class.

Reply:
Good suggestion.
I don't know, which device does have gpib, but in the end, it does not matter for the sake of the example.

Reply:
I found a multimeter, such that you may measure the output 😁